app/reward_mapping.py

```diff
@@ -0,0 +1,16 @@
+"""This module contains logic needed for mapping user actions to rewards"""
+
+
+def ua_to_reward_id(_):
```

Comment:
I know that this argument is a stub, but nevertheless it probably should be named more appropriately